### PR TITLE
Grant visibility of PILs to NTCOs at holding establishment

### PR DESCRIPTION
**Pull request by lennym**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -32,8 +32,8 @@ module.exports = {
       roles: ['asru:*', 'establishment:admin']
     },
     pil: {
-      list: ['asru:*', 'establishment:admin', 'establishment:read'],
-      read: ['pil:own', 'asru:*', 'establishment:admin', 'establishment:read'],
+      list: ['asru:*', 'establishment:admin', 'establishment:read', 'holdingEstablishment:role:ntco'],
+      read: ['pil:own', 'asru:*', 'establishment:admin', 'establishment:read', 'holdingEstablishment:role:ntco'],
```
Comment (joefitter) [inline config.js]:
```suggestion
      read: ['pil:own', 'asru:*', 'establishment:admin', 'establishment:read', 'establishment:role:ntco'],
```

Comment (joefitter) [reply config.js]: establishment role permissions are scoped to the establishment passed in, I don't think holdingEstablishment:role is a thing

Comment (lennym) [reply config.js]: It's probably fine to let them view everyone's PILs isn't it?

Do we need the same change on the `list` permission too? (not that I think that's actually a thing)

File: config.js [hunk 2]
```diff
@@ -32,8 +32,8 @@ module.exports = {
       roles: ['asru:*', 'establishment:admin']
     },
     pil: {
-      list: ['asru:*', 'establishment:admin', 'establishment:read'],
-      read: ['pil:own', 'asru:*', 'establishment:admin', 'establishment:read'],
+      list: ['asru:*', 'establishment:admin', 'establishment:read', 'holdingEstablishment:role:ntco'],
```
Comment (lennym) [inline config.js]:
```suggestion
      list: ['asru:*', 'establishment:admin', 'establishment:read', 'establishment:role:ntco'],
```

**Pull request merged**